nbdime/args.py

```diff
@@ -532,6 +538,7 @@ def args_for_server(arguments):
                 workdirectory='cwd',
                 base_url='base_url',
                 hide_unchanged='hide_unchanged',
+                show_base='show_base',
```

Comment:
This is mainly a convenience for shared args. For a one-off like `show_base`, it is probably better to add it manually in nbdime/webapp/nbmergeweb.